src/client/components/Registration.jsx

```diff
@@ -0,0 +1,28 @@
+const Registration = () => {
+
+    return( 
+        <main className="registrationPage">
+            <div id="registrationFormDiv">
+                <div id="registrationSell">
+                    <h2>Create your account</h2>
+                    <ul>
+                        <li>Rate your favourite Pokémon</li>
+                        <li>Discuss your ratings</li>
+                        <li>Pick your dream team</li>
+                    </ul>
+                </div>
+                <form id="registrationForm">
+                    <label htmlFor="emailR" className="emailLabelR">Email</label>
```

Comment:
What is the purpose of the "htmlFor" attribute?

Reply:
React prefers "htmlFor" instead of just "for" to ensure it doesn't get confused with the javascript for